src/Provider/IronMqProvider.php

```diff
@@ -46,7 +46,7 @@ class IronMqProvider extends AbstractProvider
     /**
      * IronMQ Queue
      *
-     * @var stdObject
+     * @var object
```

Comment:
`stdObject` doesn't exist, only `stdClass`. `object` is a typeint (see [PHP doc object](http://php.net/manual/en/language.types.object.php)).